operator/README.md

```diff
@@ -0,0 +1,175 @@
+| nodeResources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | Describes the compute resource requirements for the pods running on node(s) |
+| registryCert | string | Encoded tls certificate signed by a certificate authority used for driver's controller registry server | generated by operator self-signed CA |
+| nodeControllerCert | string | Encoded tls certificate signed by a certificate authority used for driver's node controllers | generated by operator self-signed CA |
+| registryKey | string | Encoded RSA private key used for signing by `registryCert` | generated by the operator |
+| nodeControllerKey | string | Encoded RSA private key used for signing by `nodeControllerCert` | generated by the operator |
+| caCert | string | Certificate of the CA by which the `registryCert` and `controllerCert` are signed | self-signed certificate generated by the operator |
+| nodeSelector | string map | [Labels to use for selecting Nodes](../README.md#run-pmem-csi-on-kubernetes) on which PMEM-CSI driver should run. | `{ "storage": "pmem" }`|
+
+<sup>1</sup> Image versions are depend on the Kubernetes cluster version.
+
+#### DeploymentStatus
+
+A PMEM-CSI Deployment's `status` field is a `DeploymentStatus` object, which has
+a `phase` field. The phase of a Deployment is high-level summary of where the
+Deployment is in it's lifecycle.
+
+The possible `phase` values and their meaning are as below:
+
+| value | meaning |
+|---|---|---|
+| Initializing | The required sub-resources of the `Deployment` are being initialized |
+| Running | All the required sub-resources of the `Deployment` are created. This does not mean that the deployed driver is fully ready to use, there might still possibility of failures. Reasons for such failures can be determined by looking into individual pod status |
+| Failed | For some reason the state of the `Deployment` failed and cannot be progressed<sup>2</sup> |
+
+<sup>2</sup> Failure reason is supposed to be carried by one of additional `DeploymentStatus` filed, but not implemented yet.
+
+### Driver certificates
+
+All [PMEM-CSI driver communication is protected by mutual
+TLS](../README.md#security) to ensure that malicious or compromised entities in
+the cluster cannot interfere with its operation. So the driver deployment needs
+to be provided with those certificates and corresponding private keys using appropriate fields in deployment specification. These encoded certificates and private keys are made available to driver pods via Kubernetes [secrets](https://kubernetes.io/docs/concepts/configuration/secret/) by the operator.
+
+If the private keys and/or certificates data is missing in a deployment, the operator generates them using a self-signed CA.
+
+**NOTE:** A production deployment is not supposed to depend on the operator's self-signed CA instead must provide the certificates generated from a trusted certificate authority.
```

Comment:
s/deployment is not/deployment that is not/

Reply:
Done.